Yosemite/Yosemite/Stores/ProductStore.swift

```diff
@@ -1099,3 +1130,9 @@ public enum ProductLoadError: Error, Equatable {
         }
     }
 }
+
+private struct ProductDetailsFromScannedTextsResponse: Decodable {
+    let name: String
+    let description: String
+    let language: String
+}
```

Comment:
Nit: May I ask why we need to have a separate struct for the response here instead of reusing `ProductDetailsFromScannedTexts`? Can we create a private extension of `ProductDetailsFromScannedTexts` that conforms to `Decodable` here?

Reply:
Like you said, I was hoping not to make the `Decodable` public knowledge. When conforming a struct to `Decodable` in an extension, an error appears:

```Extension outside of file declaring struct 'ProductDetailsFromScannedTexts' prevents automatic synthesis of 'init(from:)' for protocol 'Decodable'```

If it makes things cleaner, I can remove the Yosemite model and just use the one from Networking conforming to Decodable.

Reply:
I realized that both structs are in the Yosemite layer, it makes sense to just use struct. Updated in https://github.com/woocommerce/woocommerce-ios/pull/10183/commits/cdb48f894eaf36bd107065c79ea2c7417c144b6b